R/utils.R

```diff
@@ -351,6 +351,16 @@ h_factor_ref_data <- function(x, ref, vars) {
   x
 }
 
+#' Obtain Right Hand Side Variables of `mmrm_tmb_formula_parts` Object
+#'
+#' @param object (`mmrm_tmb_formula_parts`)\cr object.
+#'
+#' @keywords internal
+h_mmrm_vars <- function(object) {
```

Comment:
nice, wondering if this could be directly a component of `mmrm_tmb_formula_parts` upon its construction? then we could just take it from there